sktime/forecasting/var.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+__all__ = ["VectorAutoRegression"]
+__author__ = ["Taiwo Owoseni"]
+
+from statsmodels.tsa.api import VAR as _VAR
+from sktime.forecasting.base.adapters import _StatsModelsAdapter
+from sktime.utils.validation.series import check_series
+from sktime.utils.validation.forecasting import check_X
+import pandas as pd
+import numpy as np
+
+DEFAULT_ALPHA = 0.05
+
+
+class VectorAutoRegression(_StatsModelsAdapter):
+    _fitted_param_names = ("aic", "fpe", "hqic", "bic")
+    _tags = {
+        "univariate-only": False,
+        "requires-fh-in-fit": False,
+        "handles-missing-data": False,
+    }
+
+    def __init__(self, dates=None, freq=None, missing="none"):
+        # Model params
+        self._y = None
+        self.dates = dates
+        self.freq = freq
+        self.missing = missing
+
+        super(VectorAutoRegression, self).__init__()
+
+    def fit(self, y, X=None, fh=None):
```

Comment:
ultimately, you don't want to have a `fit` here.

Can you explain which part you really need?

Reply:
I think it is necessary to have a fit method here. I need it to override the baseforecaster's fit method. The Baseforecaster's fit method checks if `y` is univariate and raises an error if it is not. Hence, the fit method. Although, I could have gotten rid of `_fit` . I thought having two fit methods would make it easier when we have to make  futures changes to `var` and  baseforecaster accept dataframes.

Reply:
yes, I see - I would suggest, let's make sure that `fit` can be easily removed once the `BaseForecaster` lets data frames through.